odoo/addons/base/migrations/12.0.1.3/pre-migration.py

```diff
@@ -1,12 +1,47 @@
-# © 2018 Opener B.V. (stefan@opener.amsterdam)
+# Copyright 2018 Opener B.V. (stefan@opener.amsterdam)
+# Copyright 2018 Paul Catinean <https://github.com/PCatinean>
+# Copyright 2018 Tecnativa - Pedro M. Baeza
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
 from odoo.addons.openupgrade_records.lib import apriori
 from openupgradelib import openupgrade
 
+model_renames = [
+    ('product.uom', 'uom.uom'),
+    ('product.uom.category', 'uom.category'),
+]
+
+table_renames = [
+    ('product_uom', 'uom_uom'),
+    ('product_uom_categ', 'uom_category'),
+]
+
 xmlid_renames = [
     ('auth_signup.default_template_user', 'base.template_portal_user_id'),
     ('auth_signup.default_template_user_config',
      'base.default_template_user_config'),
+    ('product.group_uom', 'uom.group_uom'),
+    ('product.product_uom_categ_unit', 'uom.product_uom_categ_unit'),
```

Comment:
Missing  `('product.product_uom_categ_vol', 'uom.product_uom_categ_vol'),`

Reply:
Can you point where in 11.0 code this record is defined?

Reply:
Is it? https://github.com/odoo/odoo/blob/45bd7a58d5202d6b9387636317eb33efa569f458/addons/product/data/product_data.xml#L15

Reply:
I mean `product_uom_categ_vol` pointed by @cuongnmtm. That one doesn't exist...

Reply:
This one: https://github.com/odoo/odoo/blob/45bd7a58d5202d6b9387636317eb33efa569f458/addons/product/data/product_data.xml#L27

Reply:
Is that v11?

Reply:
Yes, that is. https://github.com/odoo/odoo/blob/11.0/addons/product/data/product_data.xml

Reply:
Thanks. Fixed in ab5e14825252bfd41d87c481d1318c148855b057